### PR TITLE
Issue911

### LINKED ISSUE #911: invisible button on itemnav
**Describe the bug**
When opening the itemnav, at the top right of the itemnav, right of the x-button, has an invisible button that closes the itemnav.

**To Reproduce**
1. Go to https://nightly.opencircuits.io/
2. Click on "+" button to open itemnav
3. Move mouse to the right side of "x" button
Screenshot (12)
Around this area
4. Observe that there is an invisible button

**Expected behavior**
I don't know if there is supposed to be a button there or not, but I think that it is a remanent of the "+" itemnav button.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by shadowluigi27**
Code disables and reenables clicking on itemnav  depending on state it is in

fixes #911

### CODE REVIEW

File: src/site/shared/containers/ItemNav/index.scss [hunk 1]
```diff
@@ -153,9 +153,10 @@ $tab-padding: 8px;
             border-color: transparent;
 
             cursor: pointer;
+            pointer-events:none;
```
Comment (TGCrystal) [inline src/site/shared/containers/ItemNav/index.scss:156]: nit: add space between `:` and `none`

File: src/site/shared/containers/ItemNav/index.scss [hunk 2]
```diff
@@ -153,9 +153,10 @@ $tab-padding: 8px;
...
 
             transition: 0.5s;
-
+
```
Comment (TGCrystal) [inline src/site/shared/containers/ItemNav/index.scss:159]: nit: remove this unnecessary indentation

File: src/site/shared/containers/ItemNav/index.scss [hunk 3]
```diff
@@ -168,7 +169,7 @@ $tab-padding: 8px;
 
             &:after {
                 content: '×';
-
+                pointer-events:all;
```
Comment (TGCrystal) [inline src/site/shared/containers/ItemNav/index.scss:172]: nit: add space between `:` and `all`

**Review by TGCrystal — CHANGES_REQUESTED** [review]

File: src/site/shared/containers/ItemNav/index.scss [hunk 4]
```diff
@@ -153,9 +153,10 @@ $tab-padding: 8px;
             border-color: transparent;
 
             cursor: pointer;
-
+            pointer-events: none;
+            
             transition: 0.5s;
-
+
```
Comment (TGCrystal) [inline src/site/shared/containers/ItemNav/index.scss:159]: nit: This indentation is still present

**Review by TGCrystal — CHANGES_REQUESTED** [review]

File: src/site/shared/containers/ItemNav/index.scss (hunk 4 above)
Comment (shadowluigi27) [reply src/site/shared/containers/ItemNav/index.scss:159]: Maybe I didn't understand from the way github shows it. Is it saying line 159 should NOT have an indent?

Comment (TGCrystal) [reply src/site/shared/containers/ItemNav/index.scss:159]: It is saying that this pr is adding the highlighted indentation. Previously, this was an empty line, now in this pr it is a line with some whitespace in it.

**Review by TGCrystal — APPROVED** [review]

File: src/site/shared/containers/ItemNav/index.scss (hunk 4 above)
Comment (shadowluigi27) [reply src/site/shared/containers/ItemNav/index.scss]: Alright, thanks! still a bit new to how GitHub shows changes

**Review by LeonMontealegre — APPROVED** [review]

**Pull request merged**